src/api/controllers/password.reset.js

```diff
@@ -55,19 +55,17 @@ export default class PasswordReset {
     const result = {};
     const userEmail = req.params.email;
     const userPassword = req.body.password;
-    const salt = genSaltSync(parseFloat(process.env.BCRYPT_HASH_ROUNDS) || 10);
+    const salt = genSaltSync(parseFloat(process.env.BCRYPT_HASH_ROUNDS));
```

Comment:
@Ramadhan0 why do you have changes in this file too?

Reply:
I worked with Premise and we decided to remove and add it in dotenv